app/controllers/admin/fixed_events_controller.rb

```diff
@@ -23,6 +23,7 @@ def edit; end
 
     def create
       @fixed_event = FixedEvent.new(fixed_event_params)
+      attach_attachments
```

Comment:
i overtook the logic from activities_controller which also saves the files even though the model is not validated. maybe we could change it for both?

Reply:
Kann man schon so machen...